docs/table_design/list_partitioning.md

```diff
@@ -0,0 +1,110 @@
+
+**List Partitioning is particularly useful for scenarios where a partition needs to include multiple values for each partition column**. For example, if a table includes the "City" column representing the city to which an individual belongs, and you often query and manage data by states and cities. At table creation, you can use the `City`  column as the partition column for List Partitioning and specify that data of various cities within the same state are placed in one partition, for example `PARTITION pCalifornia VALUES IN ("Los Angeles", "San Francisco", "San Diego")`. This method can accelerate queries based on states and cities while facilitating data management.
+
+If a partition only needs to contain data with the same value of each partition column, it is recommended to use [expression partitioning](./expression_partitioning.md).
+
+**Comparison between list partitioning and [expression partitioning](./expression_partitioning.md)**
+
+The main difference between List partitioning and Expression partitioning (recommended) is that List partitioning requires you to manually create partitions one by one. On the other hand, Expression partitioning can automatically create partitions during loading to simplify partitioning. And in most cases, Expression partitioning can replace List partitioning. The specific comparison between the two is shown in the following table:
+
+| Partitioning method                                      | **list partitioning**                                        | E**xpression partitioning**                                  |
+| -------------------------------------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| Syntax                                                   | `PARTITION BY LIST (partition_columns)（    PARTITION <partition_name> VALUES IN (value_list)    [, ...] )` | `PARTITION BY <partition_columns>`                           |
+| Multiple values for each partition column in a partition | Supported. A partition can store data with different values in each partition column. In the following example, when the loaded data with  values `Los Angeles`, `San Francisco`, and `San Diego` in the the `city` column, all the data is stored in one partition.<br>`pCalifornia`.`PARTITION BY LIST (city) (    PARTITION pCalifornia VALUES IN ("Los Angeles","San Francisco","San Diego")    [, ...] )` | Not supported. A partition stores data with the same value in  the partition column.For example, the expression `PARTITION BY (city)` is used in expression partitioning. When the loaded data contains values `Los Angeles`, `San Francisco`, and `San Diego` in the `city` column, StarRocks will automatically create three partitions `pLosAngeles`, `pSanFrancisco`, and `pSanDiego`. The three partitions respectively store data with values `Los Angeles,` `San Francisco`, and `San Diego` in the `city` column. |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/acc2c8f9-358b-4625-826c-89deda93c0c9)
A space is required.